rgw/v2/tests/s3_swift/configs/test_lc_with_prefix_dot.yaml

```diff
@@ -0,0 +1,19 @@
+# script: test_bucket_lifecycle_object_expiration.py
+# CEPH-11192 - Test object prefixes with delimiter '/', '_', '-', '.'
+config:
```

Comment:
Please add bucket count to more than 2 in all config file

Reply:
done